workspace/TS100/Core/Src/OLED.cpp

```diff
@@ -62,6 +63,14 @@ uint8_t OLED_Setup_Array[] = {
 const uint8_t REFRESH_COMMANDS[17] = { 0x80, 0xAF, 0x80, 0x21, 0x80, 0x20, 0x80,
 		0x7F, 0x80, 0xC0, 0x80, 0x22, 0x80, 0x00, 0x80, 0x01, 0x40 };
 
+static uint8_t easeInOutTiming(uint8_t t) {
```

Comment:
Could you add a comment on what these two are for rather than magic math equations?

Reply:
Sure, I added some documentation :)